StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1012OpeningCurlyBracketsMustBeSpacedCorrectly.cs

```diff
@@ -111,6 +112,12 @@ private void HandleOpenBraceToken(SyntaxTreeAnalysisContext context, SyntaxToken
             followedBySpace = token.HasTrailingTrivia;
             lastInLine = followedBySpace && token.TrailingTrivia.Any(SyntaxKind.EndOfLineTrivia);
 
+            if (token.Parent as InterpolationSyntax != null)
```

Comment:
Wow. I thought it was the other way around. You learn something everyday. I changed that.